demo/dlb-init.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -eu
```

Comment:
btw, do we really need to use bash here? Can we modify the script to work on sh?

Reply:
As this is toybox, we will not get neither real bash or sh but toybox implemented shell, so I guess this line does not change what we get. But we can change and see. In contents, seems we dont have bash-specifics so it is ok to use sh.

Reply:
checkbashisms seems to be happy about it:
```
$ checkbashisms /tmp/dlb-init.sh 
could not find any possible bashisms in bash script /tmp/dlb-init.sh
```